### DISCUSSION TITLE
Install macOS Issue - stuck on Apple Xcode install

### DISCUSSION THREAD

**Discussion by kbejado** [thegetty/quire#842] [Get Help]
Hello, I am new to using Quire so I have been attempting to install it on my macOS laptop. However, I canʻt get past the 1st step which is to install Apple’s Xcode. I am able to enter the command on my terminal and press the ʻinstallʻ tab. However, it seems to be stuck on the ʻfinding softwareʻ window and the progress bar does not move up. 

Any help or direction would be greatly appreciated!

**Comment by Erin-Cecele** [comment]
Hi @kbejado. I would try proceeding without Apple Xcode. Depending on the system, it is not always necessary. Just let us know if you receive any error messages moving forward, and we can troubleshoot from there.

**Comment by kbejado** [comment]
Hi @Erin-Cecele, I am receiving these error messages. Please see attached.
<img width="567" alt="Screen Shot 2023-09-12 at 7 53 48 AM" src="https://github.com/thegetty/quire/assets/144129236/473f1a4b-e4a0-4f07-9e4c-e2169266ba53">

**Comment by Erin-Cecele** [comment]
No problem, per our [Troubleshooting instructions](https://quire.getty.edu/docs-v1/troubleshooting/#installing-quire-on-macos):

If you receive an error message that says “Error: EACCES: permission denied,” this is related to the ownership and permissions settings for the global node_modules directories that are installed with Quire. To fix this issue, run the following command and then try re-installing Quire:

```
sudo chown -R $(whoami) $(npm config get prefix)/{lib/node_modules,bin,share}
```

**Comment by kbejado** [comment]
Hi! It is now asking for a password but wonʻt let me type. 
<img width="562" alt="Screen Shot 2023-09-12 at 1 44 35 PM" src="https://github.com/thegetty/quire/assets/144129236/7de6da92-3fdb-4fa1-bb90-ef98712ee89d">

**Comment by Erin-Cecele** [comment]
@kbejado you won't be able to see what you type, but it's there. Just type your password as normal and hit enter.

**Reply by kbejado** [reply]
@Erin-Cecele I am receiving this error message now. 
<img width="754" alt="Screen Shot 2023-09-12 at 1 49 19 PM" src="https://github.com/thegetty/quire/assets/144129236/2a3fd4c4-d451-4ba6-8c32-a26d0366a053">

**Comment by Erin-Cecele** [comment]
Click on "Go" at the top of your Mac screen and then click "Go To Folder" 
<img width="506" alt="Screen Shot 2023-09-12 at 10 53 56 AM" src="https://github.com/thegetty/quire/assets/62075223/1c2edf27-f3b5-4744-9ca6-b12d838fe160">

Then type `/usr/local/lib/node_modules/@thegetty/quire-cli` and delete the `quire-cli` folder.

<img width="447" alt="Screen Shot 2023-09-12 at 10 56 39 AM" src="https://github.com/thegetty/quire/assets/62075223/31ecd732-3db2-424c-88cf-e391d3baadde">

The first time you tried to install it started to download quire onto your computer and now that it's trying again it's getting stuck because it thinks quire is already there. 

Once you've deleted the folder, try the install process again.

**Reply by kbejado** [reply]
I see! Would I delete all of these folders? 
<img width="778" alt="Screen Shot 2023-09-12 at 2 00 46 PM" src="https://github.com/thegetty/quire/assets/144129236/8b9730fe-fc05-4eeb-bef6-95016d1d1bbf">

**Comment by Erin-Cecele** [comment]
Yes, just delete the entire `quire-cli` folder. Thank you!

**Comment by Erin-Cecele** [comment]
@kbejado how are things coming along?
